config.go

```diff
@@ -362,7 +364,8 @@ func loadConfig() (*config, []string, error) {
 	if !(preCfg.SimNet) || preCfg.ConfigFile != defaultConfigFile {
 		err := flags.NewIniParser(parser).ParseFile(preCfg.ConfigFile)
 		if err != nil {
-			if _, ok := err.(*os.PathError); !ok {
+			var e *os.PathError
+			if !errors.As(err, &e) {
```

Comment:
Can use Is since we don't care about the unwrapped value.

Reply:
We care about the type, so we use As().  Is() matches values.